test/lightning/invocation_test.exs

```diff
@@ -1125,4 +1128,37 @@ defmodule Lightning.InvocationTest do
       %{id: Ecto.UUID.generate(), message: message, timestamp: build(:timestamp)}
     end
   end
+
+  describe "export_workorders/3" do
+    setup do
+      project = insert(:project)
+      user = insert(:user)
+      search_params = SearchParams.new(%{})
+
+      {:ok, project: project, user: user, search_params: search_params}
+    end
+
+    test "initiates a work orders export successfully and logs the audit event",
+         %{
+           project: project,
+           user: user,
+           search_params: search_params
+         } do
+      Oban.Testing.with_testing_mode(:manual, fn ->
```

Comment:
@elias-ba Is there value in asserting that the Oban job has been correctly enqueued?

Reply:
This is the best solution I found to test `export_workorders` function without having Oban to execute the job behind the scenes.

Reply:
@elias-ba Apologies, I worded that poorly. What I should have said is "How do you feel about adding an `assert_enqueued` to this test to validate the the Oban job has been correctly enqueued?